test/tests/providers/global/platform/atlas/alerting-rules/alloy.rules.test.yml

```diff
@@ -225,3 +225,37 @@ tests:
               dashboard: "53c1ecddc3a1d5d4b8d6cd0c23676c31/alloy-logs-overview"
               description: "Scraping of all logging-agent pods to check if one failed every 30 minutes."
               opsrecipe: "alloy/"
+
+  # Test MonitoringAgentDown
```

Comment:
The most interesting / usual MonitoringAgentDown case is when there is no `up` data. Because if there's no monitoring agent, there's no data.
So I think it makes sense to have the "no data" phase longer, so it triggers the alert (meaning more than 20min).
Also, we could check the InhibitionMonitoringAgentDown here as well, with the same set of metrics.

Reply:
TBH I copied the tests for PrometheusAgentFailing alert but i'm open for improvements ideas

Reply:
I like it better like that! Thanks! :+1: